patroni/validator.py

```diff
@@ -392,6 +392,37 @@ def __init__(self, name: str, default: OptionalType[Any] = None) -> None:
         self.default = default
 
 
+class OneOf(object):
+    """Mark that at most one option from a `Case` can be suplied
```

Comment:
```suggestion
"""Mark that at most one option from a :class:`Case` can be suplied
```

(If you mean the class)